modules/client-configuration/pages/supported-features.adoc

```diff
@@ -41,114 +41,114 @@ The operating system you run on a client is supported by the organization that s
 | Salt Clients
 
 | {sle} 15
-| {x86}_64, {ppc}, {ibmz}, ARM
+| {x86_64}, {ppc}, {ibmz}, ARM
```

Comment:
Changes in this file seem unrelated to Debian client updates ;)

Reply:
yeah, that's why I noted it as a drive-by edit in the description :wink: